decidim-core/db/migrate/20220204121246_change_uppercased_nicknamed.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class ChangeUppercasedNicknamed < ActiveRecord::Migration[6.0]
+  def up
+    # list of users already changed in the process
+    has_changed = []
```

Comment:
Here I think we can use a easier way to retrieve duplicated nicknames see 

`Decidim::User.where("nickname ILIKE ?", nickname.downcase).order(:created_at)`, it returns a list of user with same nickname and is case insensitive

I remain available if needed